### ISSUE TITLE
Check for issue with long `delayTootsSameItem` if `ignoreFirstRun` is `true`

### ISSUE THREAD

**Issue by nhoizey** [nhoizey/github-action-feed-to-mastodon#61]
When `ignoreFirstRun = true`, the first run doesn't create any toot, but stores the timestamp in each item and a text message in the list of toots:

```json
{
  "https://nicolas-hoizey.photo/galleries/travels/europe/italy/venice/burano/colourful-burano/": {
    "id": "https://nicolas-hoizey.photo/galleries/travels/europe/italy/venice/burano/colourful-burano/",
    "url": "https://nicolas-hoizey.photo/galleries/travels/europe/italy/venice/burano/colourful-burano/",
    "language": "en",
    "title": "Colourful Burano",
    "date_published": "2008-04-26T14:37:21Z",
    "content_text": "“Colourful Burano”\n\n\n\n🔎 https://nicolas-hoizey.photo/galleries/travels/europe/italy/venice/burano/colourful-burano/\n\n📅 26 April 2008\n\n📸 Canon 350D + 17-85mm\n🎛️ ISO 100, ƒ/10, 1/200 s\n\n#Travel #TravelPhotography #Europe #Italy #Venice #Burano #Photography",
    "attachments": [
      {
        "url": "https://nicolas-hoizey.photo/photos/colourful-burano/small.jpg",
        "mime_type": "image/jpeg",
        "title": "Colourful Burano"
      }
    ],
    "toots": [
      "Ignored during first run (see `ignoreFirstRun` input)"
    ],
    "lastTootTimestamp": 1704500964402
  }
}
```

Which means no new toot will be sent:
- ~~ever if `nbTootsPerItem = 1`?~~ -> this was on purpose 🤦
- until a potentially long `delayTootsSameItem` if `nbTootsPerItem > 1` AND there is no new content

**Comment by nhoizey** [comment]
Additional issue:

When new items are added to the feed later:
- they will have 0 toots,
- so will be sent to Mastodon in priority,
- so will know have 1 toot,
- so will have the same priority as items from the initial feed, which have 1 fake toot
- so will sometimes be tooted once again sooner that what would be preferable

**Comment by nhoizey** [comment]
Issue with `nbTootsPerItem: 1` fixed in 92aced79208d01888691ef78dbc7a575ef11c186

**Comment by nhoizey** [comment]
> Issue with `nbTootsPerItem: 1` fixed in [92aced7](https://github.com/nhoizey/github-action-feed-to-mastodon/commit/92aced79208d01888691ef78dbc7a575ef11c186)

This was an error.

**Comment by nhoizey** [comment]
No issue after all… 😅
